media-sound/audacity/audacity-3.2.2.ebuild

```diff
@@ -0,0 +1,234 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+WX_GTK_VER="3.2-gtk3"
+
+inherit cmake wxwidgets xdg
+
+MY_P="Audacity-${PV}"
+DOC_PV="${PV}"
+DESCRIPTION="Free crossplatform audio editor"
+HOMEPAGE="https://web.audacityteam.org/"
+
+# A header-only thread pool library, without a build system, about 100
+# lines of code.  Probably not worth packaging individually.  Check
+# cmake-proxies/CMakeLists.txt and search for "ThreadPool".
+MY_THREADPOOL_DATE=20140926
+MY_THREADPOOL="https://raw.githubusercontent.com/progschj/ThreadPool/9a42ec1329f259a5f4881a291db1dcb8f2ad9040/ThreadPool.h -> progschj-ThreadPool-${MY_THREADPOOL_DATE}.h"
+SRC_URI="https://github.com/audacity/audacity/archive/${MY_P}.tar.gz
+	doc? ( https://github.com/audacity/audacity-manual/releases/download/v${PV}/audacity-manual-${PV}.tar.gz )
+	telemetry? ( ${MY_THREADPOOL} )
+	audiocom? ( ${MY_THREADPOOL} )"
+S="${WORKDIR}/${PN}-${MY_P}"
+
+LICENSE="GPL-2+"
```

Comment:
Please re-check, 
https://github.com/audacity/audacity/blob/master/README.txt
at least the *doc?* part should apply here if everything else is unbundled. 

Also check the LICENSE for Threadpool, 
https://github.com/progschj/ThreadPool/

Reply:
Ah, right, doc and threadpool, forgot about those.  The GPL2 part itself comes from the LICENSE.txt and the fact that I couldn't find any files specifying other GPL versions (despite LICENSE.txt saying there could be some, and there being a copy embedded in the about dialog).  I must've forgotten to add GPL-3+ after finding that copy because I was working on the technical bits.

Regardless, will recheck